### DISCUSSION TITLE
SRSRAN CU/CP DPDK F1/N2/N3 and ZeroMQ Clarification

### DISCUSSION THREAD

**Discussion by infinitydon** [srsran/srsRAN_Project#648] [General Help] [answered]
Hi,

I will like to confirm the following:

- Does CU/CP configuration support using separate DPDK interfaces for the F1, N2 and N3
- Can we still use ZeroMQ for simulation when using the DPDK mode in the CU/CP?

Thanks

**Comment by andrepuschmann** [comment]
DPDK is currently only used for open front haul. So in a zmq setup it doesn't play a role. Also note that dpdk is a offloading technique for workloads that require high data/packet rates so no unlikely to be required for zmq use cases anyway.

**Reply by infinitydon** [reply]
If the fronthaul is using userplane acceleration only, will the N3 not be a bottleneck if it is not using DPDK also?

**Reply by andrepuschmann** [reply, marked as answer]
So far it's not a bottleneck for single cell, even at 1.5Gpbs. But we are prepared to use eBPF or even DPDK on N3 and/or F1u if it's required.
